src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,93 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import classname from 'classnames';
+import { getMovie } from '../../api/api';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+interface Props {
+  addMovies: (value: Movie) => void
+}
+
+export const FindMovie: React.FC<Props> = ({ addMovies }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const findMovie = () => {
+    getMovie(title)
+      .then(response => setMovie(response))
+      .catch(() => console.log('film not found, write correct title'));
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classname('input',
+                { 'is-danger': !movie, 'is-primary': movie })}
+              value={title}
+              onChange={(event) => (
+                setTitle(event.target.value)
+              )}
+            />
+          </div>
+
+          <p className="help is-danger">
+            {!movie ? 'Can`t find a movie with such a title' : ''}
+          </p>
```

Comment:
This paragraph should be visible only if movie wasn't found

Reply:
agree